specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ManagedEnvironments_CustomInfrastructureResourceGroup_Create.json

```diff
@@ -0,0 +1,147 @@
+{

```

Comment:
Why have exclusive example vs add to `specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ManagedEnvironments_CreateOrUpdate.json`? General example pattern I have seen is ResourceType_Operation.

Reply:
I updated the original example to show that the property is returned with a default value when not provided. The new example file shows using a custom value for the property. AFAIK there is no restriction on having several example files for different scenarios.